### PR TITLE
EndToEnd Module name change

### PR DESCRIPTION
**Pull request by SuyDesignz**
@jplag/maintainer @jplag/studdev 
This PR contains the renaming of the `endToEndTesting` module to `endtoend-testing` and the renaming of the packages from `end-to-end-testing` to `endtoend`.

The further info can be taken from the isuee [Naming issues with the end-to-end module](https://github.com/jplag/JPlag/issues/605)

### CODE REVIEW

File: jplag.cli/pom.xml [hunk 1]
```diff
@@ -72,9 +72,8 @@
         </dependency>
         <dependency>
             <groupId>de.jplag</groupId>
-            <artifactId>endToEndTesting</artifactId>
+            <artifactId>endtoend-testing</artifactId>
```
Comment (dfuchss) [inline jplag.cli/pom.xml:75]:
```suggestion
            <artifactId>endtoend-testing</artifactId>
            <scope>test</scope>
```

**Review by dfuchss — APPROVED** [review]

**Comment by SuyDesignz** [comment]
the sonarcloud is currently blocking the merge because of the test coverage. is this due to the changed scope?
``` xml
            <artifactId>endtoend-testing</artifactId>
            <scope>test</scope>
```

**Comment by dfuchss** [comment]
Yes, but thats not a problem. I've cleaned up the dependency since the CLI should not have a dependency to e2e test. I'll merge the PR soon

**Comment by SuyDesignz** [comment]
I just saw in the build output that a few classes still have no description. should I change this now in this PR or create an extra one for it?

**Comment by dfuchss** [comment]
Please wait 5 min (that's the approx time I need to merge :) ). Afterwards, open another PR for that.

**Pull request merged**